### PR TITLE
[HTD-38] Feat: Add `Integration grid` component

### PR DESCRIPTION
**Pull request by RBoraurito**
## Assets

| before | after |
| ------ | ----- |
|        |       |

## Before Merge Checklist

The following steps were checked before merging a PR

- [x] checked and fixed code style issues
- [x] checked and removed unnecessary/junky code
- [x] manually tested introduced solution
- [ ] added needed tests to automate testing this code in the future

### CODE REVIEW

File: src/components/sections/integration/grid/index.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { StyledIntegrationGrid } from './grid.styles';
+
+export const IntegrationGrid = ({ children }: Component) => {
+  return <StyledIntegrationGrid>{children}</StyledIntegrationGrid>;
+};
```
Comment (nephlin7) [inline src/components/sections/integration/grid/index.tsx]: Do you think we can make this component receive an array of integrations instead? Something like:

```suggestion
type IntegrationGridProps = {
  integrations: IntegrationCardProps[];
};

export const IntegrationGrid = ({ integrations }: Component<IntegrationGridProps>) => {
  return (
    <StyledIntegrationGrid>
      {Array.isArray(integrations) &&
        integrations.length > 0 &&
        integrations.map(integration => <IntegrationCard {...integration} />)}
    </StyledIntegrationGrid>
  );
};
```

File: src/components/sections/integration/grid/index.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import type { IntegrationGridProps } from './grid.types';
+
+import { IntegrationCard } from '@/components/modules/integration-card';
+
+export const IntegrationGrid = ({ integrations }: IntegrationGridProps) => {
+  return (
+    <StyledIntegrationGrid>
+      {Array.isArray(integrations) &&
+        integrations.length > 0 &&
+        integrations.map(card => <IntegrationCard {...card} />)}
```
Comment (nephlin7) [inline src/components/sections/integration/grid/index.tsx]: Adds the missing key to the map, please

**Review by nephlin7 — APPROVED** [review]
@RBoraurito, check the last comment before merge, please.

File: src/components/sections/integration/grid/index.tsx (hunk 2 above)
Comment (RBoraurito) [reply src/components/sections/integration/grid/index.tsx]: I added the title as key, for this example is going to be repeated but with actual data shouldn't be.

**Pull request merged**